includes/Api/ApiViewDumps.php

```diff
@@ -77,22 +77,11 @@ private function getDownloadUrl( $config, $dump ) {
 			return $dump->dumps_filename;
 		}
 
-		// If wgDataDumpDownloadUrl is configured, use that
```

Comment:
This should not be removed. It means we are using php which reduces capacity & slowness & possible memory issues.

Reply:
> This should not be removed. It means we are using php which reduces capacity & slowness & possible memory issues.

It cannot stream chunks if we use this so it has to be removed, additionally I actually notice it works a lot faster by using the internal streamer instead...